aws/resource_aws_glue_crawler.go

```diff
@@ -459,6 +463,14 @@ func resourceAwsGlueCrawlerRead(d *schema.ResourceData, meta interface{}) error
 		return nil
 	}
 
+	arn := arn.ARN{
```

Comment:
I would be careful to not shadow the import name by calling this `crawlerARN :=` or something. 👍

Reply:
Thanks, fixed.